src/main/java/de/retest/recheck/ui/TargetFinder.java

```diff
@@ -18,6 +18,8 @@
 
 	private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger( TargetFinder.class );
 
+	static final double ELEMENT_MATCH_THRESHOLD = RecheckProperties.getInstance().elementMatchThreshold();
```

Comment:
If we use static here, the properties might not yet be initialized. Thus only the default value will be used. Please check this and cleanup the other instances.

Reply:
Works as expected, also in other places. `init()` is called beforehand.